pkg/repository/postgres/tenant.go

```diff
@@ -85,6 +94,23 @@ func (r *tenantAPIRepository) CreateTenant(ctx context.Context, opts *repository
 		return nil, err
 	}
 
+	if user != nil {
+		// add the user as an owner of the tenant
+		_, err = r.CreateTenantMember(ctx, tenantId, &repository.CreateTenantMemberOpts{
```

Comment:
it makes more sense to me to create a tenant owner on create, without this the hook is racy